Services/Simulations.cs

```diff
@@ -21,6 +21,9 @@ public interface ISimulations
         // Get list of simulations.
         Task<IList<Models.Simulation>> GetListAsync();
 
+        // Get list of simulations with statistics.
+        Task<IList<Models.Simulation>> GetListWithStatisticsAsync();
```

Comment:
Please add a comment explaining that GetList has better performance than calling GetListWithStatisticsAsync.

Reply:
added